lingetic-nextjs-frontend/app/languages/[language]/EmptyState.tsx

```
@@ -0,0 +1,30 @@
+      <div className="bg-white p-8 rounded-xl shadow-lg max-w-md w-full text-center">
+        <BookOpen className="h-12 w-12 text-[#2563eb] mx-auto mb-4" />
+        <h2 className="text-2xl font-bold text-[#374151] mb-2">
+          No Question Lists Available
+        </h2>
+        <p className="text-[#374151] mb-6">
+          {`We don't have any question lists for ${language} yet. Please check back later.`}
+        </p>
+        <button
+          onClick={() => router.push("/languages")}
+          className="bg-[#2563eb] text-white px-6 py-3 rounded-lg font-medium hover:bg-blue-700 transition-colors"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)